release-packaging/HelpSource/Classes/FluidDataSetWr.schelp

```diff
@@ -132,18 +112,24 @@ OSCFunc({
 ~ds.print;
 ~ds.clear
 
-// circular writing
+::
+strong::circular writing::
+Each time link::Classes/FluidDataSetWr:: is triggered it is like the link::Classes/FluidDataSet#-setPoint:: method so if the identifier does not exist it creates it. If the identifier does it exist then it updates it with the new values.
```

Comment:
Too many `it`s! Suggest 

'so if the identifier does not exist, it creates a new point. If the identifier does exist, the corresponding point is updated'